build.sbt

```diff
@@ -20,12 +20,16 @@ scalacOptions := Seq(
   "-feature",
   "-language:reflectiveCalls"
 )
-scalacOptions in Scapegoat += "-P:scapegoat:overrideLevels:TraversableHead=Warning:OptionGet=Warning"
+scalacOptions in Scapegoat += "-P:scapegoat:overrideLevels:TraversableHead=Warning:OptionGet=Warning:TraversableLast=Warning"
```

Comment:
The goal is to address all of the errors flagged by Scapegoat, but it's ok for now - I'll raise another issue to address that separately.

Reply:
yeah of course, if not what would be the case of having scapegoat in a first place.
I would be happy to help removing some of those errors, I will raise a PR whenever I remove all I could.